docs/source/_json/addons_get.resp

```diff
@@ -0,0 +1,36 @@
+{
+  "description": "Optional plone.session refresh support.",
+  "profile_type": "default",
+  "upgrade_profiles": {},
+  "uninstall_profile_id": "plone.session:uninstall",
+  "other_profiles": [],
+  "is_installed": "FALSE",
+  "id": "plone.session",
+  "install_profile": {
+    "product": "plone.session",
+    "description": "Optional plone.session refresh support.",
+    "for": "NULL",
+    "title": "Session refresh support",
+    "pre_handler": "NULL",
+    "version": "1001",
+    "path": "/applicationPath/api/eggs/plone.session-3.7.0-py2.7.egg/plone/session/profiles/default",
```

Comment:
Do we really want to expose (absolute) filesystem paths? (Event though the endpoint is protected with `ManagePortal`)